extensions/wikia/Email/Controller/FounderController.class.php

```diff
@@ -0,0 +1,211 @@
+<?php
+
+namespace Email\Controller;
+
+use Email\Check;
+use Email\EmailController;
+use Email\Fatal;
+
+abstract class FounderController extends EmailController {
```

Comment:
Looks good. The last thing is I'd been putting the category names in the TrackingCategories class. These should probably either be part of that class, or we could do away with that class and just hardcode the categories as part of each of the controllers. My vote would be for the former.